tests/02_eradiate/01_unit/experiments/test_dem.py

```diff
@@ -0,0 +1,282 @@
+import mitsuba as mi
+import numpy as np
+import pytest
+
+import eradiate
+from eradiate import unit_registry as ureg
+from eradiate.contexts import KernelDictContext
+from eradiate.experiments._dem import DEMExperiment
+from eradiate.experiments._helpers import measure_inside_atmosphere
```

Comment:
You're already testing this function in `test_atmosphere.py`. Since `measure_inside_atmosphere()` doesn't belong to the tested module, please create a new `test_helpers.py` test file for it and move the test there. And please delete the duplicated test.